lib/hawkular/inventory/inventory_api.rb

```diff
@@ -429,6 +429,17 @@ def create_metric_type(feed_id, metric_type_id, type = 'GAUGE', unit = 'NONE', c
       MetricType.new(new_mt)
     end
 
+    # List operation types for given resource type
+    # @param [String] resource_type_path canonical path of the resource type entity
+    # @return [Array<String>] List of operation type ids
+    def list_operation_types(resource_type_path)
```

Comment:
Should we rename this to `list_operation_definitions` to make it more clear what it is - even if they are called differently in inventory?